blueflood-elasticsearch/src/main/java/com/rackspacecloud/blueflood/io/ElasticIO.java

```diff
@@ -132,7 +122,7 @@ private static String getUnit(Metric metric) {
         return metric.getUnit();
     }
 
```

Comment:
Oops, method below should have stayed private.  I had to made it internal because I was trying to extend the ElasticIO class for EnumElasticIO, which turned out to be really hard to set up testing for.  I will make this private again in the next commit.